### ISSUE TITLE
Issues with Cycle View Helper docs

### ISSUE THREAD

**Issue by divix1988** [zendframework/zend-view#93]
Can someone explain to me what is this: {{{PHP2}}} ?

Also later in the same doc, there is section about:

> You can also cycle in reverse, using the `prev()` method instead of `next()`:

However in the example there is neither a `prev()` nor `next()` method used.

```
<table>
    <?php foreach ($this->books as $book): ?>
    <tr style="background-color: {{{PHP2}}}">
       <td><?php echo $this->escapeHtml($book['author']) ?></td>
    </tr>
    <?php endforeach ?>
</table>
```

How Cycle plugin works then?

**Comment by froschdesign** [comment]
> Can someone explain to me what is this: {{{PHP2}}} ?

An error in the rendering of markdown. [Compare with with the source file.](https://github.com/zendframework/zend-view/blob/master/doc/book/helpers/cycle.md)

> However in the example there is neither a prev() nor next() method used.

The same error as above.

**Comment by weierophinney** [comment]
@froschdesign So, I wasn't entirely sure I understood what was happening, and just tried rendering... and now I see the problem: it appears that PHP code nested inside quotes is getting replaced with a placeholder — or causing errors.

When I tried with version mkdocs 0.17.3, docs built fine, but had the placeholder replacement.

When I tried with 1.0.4, I got an error: `Config value: 'markdown_extensions'. Error: can't set attribute`. I'm not sure if this is related to the markdown in the Cycle docs, or if v1 cannot handle some config option in `mkdocs.yml`.

How should we proceed?

For now, I'm clearing the milestone; when we have a fix, we can deploy at any time.
